app/src/pages/common/settingsPage/notificationsTab/notificationsTab.jsx

```diff
@@ -225,9 +226,13 @@ export class NotificationsTab extends Component {
       .map((fieldKey) => {
         const fieldInfo = ruleFieldsConfig[fieldKey];
         const fieldData = notification[fieldKey];
+        const keyVars =
+          fieldInfo && fieldInfo.getTitleVariables
+            ? fieldInfo.getTitleVariables(notification)
+            : null;
 
         return {
-          key: fieldInfo ? formatMessage(fieldInfo.title) : null,
+          key: fieldInfo ? formatMessage(fieldInfo.title, keyVars) : null,
```

Comment:
I guess we should show `AND` by default for old notification rules where the new operator field doesn't present.

![image](https://user-images.githubusercontent.com/31861805/197963403-647e3f09-de3c-4033-81d4-1eb7cdd716cc.png)